soc/arm/nordic_nrf/Kconfig.peripherals

```
@@ -115,6 +115,9 @@ config HAS_HW_NRF_QSPI
 config HAS_HW_NRF_RADIO_BLE_2M
 	bool
 
+config HAS_HW_NRF_RADIO_HIGH_TX_PWR
```

```suggestion
config HAS_HW_NRF_RADIO_TX_PWR_HIGH
```

Hope this is not nitpicking, but going by the style of other options... _BLE_2M, _BLE_CODED, _RTC0, _RTC1, I would keep _HIGH (, _MID, _LOW maybe in future) as suffix.

---

Ok, changed.